MusinDaniilTask1/ComplexNumber.cs

```diff
@@ -0,0 +1,71 @@
+﻿
+
+
+namespace MusinDaniilTask1
+{
+    public struct Complex
+    {
+        public readonly double Re;
+        public readonly double Im;
+        public readonly double Abs;
+
+        public Complex(double re = 0, double im = 0)
+        {
+
+            Re = re;
+            Im = im;
+            Abs = Math.Sqrt((re * re) + (im * im));
+        }
+        public override int GetHashCode()
+        {
+            return ToString().GetHashCode();
+        }
```

Comment:
Можно уменьшить инфраструктурный код
```suggestion
        public override int GetHashCode() => ToString().GetHashCode();
        
```